packages/devtools_app/lib/src/screens/memory/framework/memory_screen.dart

```diff
@@ -38,16 +40,27 @@ class MemoryBody extends StatefulWidget {
   MemoryBodyState createState() => MemoryBodyState();
 }
 
-class MemoryBodyState extends State<MemoryBody> {
+class MemoryBodyState extends State<MemoryBody>
+    with ProvidedControllerMixin<MemoryController, MemoryBody> {
   @override
   void initState() {
     super.initState();
     ga.screen(MemoryScreen.id);
```

Comment:
add a listener for offline mode:
`addAutoDisposeListener(offlineController.offlineMode);`